### PR TITLE
Adds figures urls, figures celery config and figures requirements

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
This PR adds:
- figures config
- figures requirements

Additionally, we need to add a feature variable `FIGURES_IS_MULTISITE` as `True`. I am guessing @NIXKnight or @jamshedali20 would add that in env.json.

### CODE REVIEW

File: lms/envs/production.py [hunk 1]
```diff
@@ -1128,3 +1128,11 @@
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+# Figures configuration
+if 'figures.apps.FiguresConfig' in INSTALLED_APPS:
+    from figures.settings.lms_production import *
+    ENV_TOKENS.setdefault('FIGURES', {})
+    update_webpack_loader(WEBPACK_LOADER, ENV_TOKENS['FIGURES'])
```
Comment (ziafazal) [inline lms/envs/production.py:1136]: Why we need to update webpack loader? we are not suppose to use figure frontend.

Comment (rehan99000) [reply lms/envs/production.py:1136]: removed

File: lms/urls.py [hunk 2]
```diff
@@ -1051,3 +1051,9 @@
 ]
 
 urlpatterns.extend(plugin_urls.get_patterns(plugin_constants.ProjectType.LMS))
+
+# Figures urls
+if 'figures' in settings.INSTALLED_APPS:
```
Comment (zubair-arbi) [inline lms/urls.py:1056]: Why we are adding this condition?

I believe we are going to install it anyway.

Comment (rehan99000) [reply lms/urls.py]: removed it. It was mentioned in the setup guide but it works without it so you are right

**Comment by rehan99000** [comment]
@zubair-arbi We don't need to add any urls or settings for figures as it does this on its own once it is installed into LMS. It's working fine for me locally as well. Following config does this in my opinion.
https://github.com/edly-io/figures/blob/1565694a3195b4c61e9f626801591046fb231323/figures/apps.py#L35

**Pull request merged**